src/components/design_system/chip/Chip.stories.tsx

```diff
@@ -0,0 +1,44 @@
+// Replace your-framework with the framework you are using, e.g. react-vite, nextjs, nextjs-vite, etc.
+import type { Meta, StoryObj } from "@storybook/react";
+
+import ContentPasteIcon from "@mui/icons-material/ContentPasteOutlined";
+import CalendarMonthOutlinedIcon from "@mui/icons-material/CalendarMonthOutlined";
+import Chip from "./Chip";
+
+const meta = {
+  title: "Components/Design System/Chip",
+  component: Chip,
+} satisfies Meta<typeof Chip>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    label: "Primary Chip",
+    className: "data-chip",
+  },
+};
+
+export const Empty: Story = {
+  args: {
+    label: "0%",
+    className: "empty-chip",
+  },
+};
+
+export const Calendar: Story = {
+  args: {
+    icon: <CalendarMonthOutlinedIcon style={{ color: "black" }} />,
```

Comment:
If `calendar` is going to be a well-defined and reusable variant of the Chip component that uses this icon, then we should have the icon be set in the component rather than forcing the developer to have to copy and paste this icon prop and import everywhere it is used.